src/runtime/containerd-shim-v2/wait.go

```diff
@@ -142,7 +142,7 @@ func watchOOMEvents(ctx context.Context, s *service) {
 
 	for {
 		select {
-		case <-ctx.Done():
+		case <-s.ctx.Done():
```

Comment:
Shouldn't this part be backported to stable-2.1 branch?

Reply:
@fidencio sure, it's better to backport to stable-2.1 after discuss above.